src/components/visualizations/chart/chartOptionsBuilder.ts

```diff
@@ -1035,11 +1071,20 @@ export function getDrillableSeries(
     });
 }
 
+function getDistinctAttributeHeaderName(result: string[], item: Execution.IResultAttributeHeaderItem): string[] {
```

Comment:
Is this unit tested? I didn't see a specific test in chartOptionsBuilder.spec.ts

Reply:
I will add unit test soon :)